ql/termstructures/yield/ratehelpers.cpp

```diff
@@ -655,6 +680,9 @@ namespace QuantLib {
     }
 
     void SwapRateHelper::setTermStructure(YieldTermStructure* t) {
+        if (swapBuiltViaGivenBuilder)
+            return;
```

Comment:
I did not see a straight forward way to access the term structure when the `MakeVanillaSwap` builder was given in the ctor. Any ideas on how we should handle this member function then?